### PR TITLE
RCAP-5848 Changes to run our plugin without the Behaviour - Background Compatibility plugin

### PR DESCRIPTION
**Pull request by prcorcoran**
Also bumped version to 2.1.0.0 and made some changes to docker-compose.yml to be able to run locally on Apple silicon and php.ini changes to be able to install the plugin.

### CODE REVIEW

**Comment by mckila** [comment]
@prcorcoran  hope you're doing well, please remove me from this repo.  (i do not have admin permission to remove myself)

thanks

Lawrence

**Comment by gary-moon** [comment]
@mckila This repository is public, so you may need to unsubscribe manually (see "Notifications" on the right side bar).

**Comment by mckila** [comment]
Thanks

File: README.md [hunk 1]
```diff
@@ -3,8 +3,8 @@
 A Joomla Extension to add LOCALiQ Tracking.
 
 Compatible with Joomla 4.0 or above.
-Tested up to: 5.0.3
-Stable tag: 2.0.0.0
+Tested up to: 5.1
+Stable tag: 2.1.0.0
```
Comment (gary-moon) [inline README.md:7]: I still see the version as `1.1.0.0`. Does the `reachedge.xml` file need to be updated as well?

image

File: README.md
Comment (gary-moon) [inline README.md:1]: The [3.8.x branch](https://github.com/reachlocal/localiq_joomla_30_extension/tree/3.8.x) is on `v1.1.0.0`, but I don't see that version on the list of [releases](https://github.com/reachlocal/localiq_joomla_30_extension/releases). Are these handled somewhere outside of GitHub?

Comment (gary-moon) [inline README.md:1]: The `zip` command in the README was not excluding the `.git` directory. I had to use the following instead:

```bash
zip -r localiq_plugin_v2.1.0.0.zip ./localiq_joomla_30_extension -x ./localiq_joomla_30_extension/.git/\*
```

Comment (prcorcoran) [reply README.md:1]: Added the v1.1.0.0 release.

Comment (prcorcoran) [reply README.md:1]: Changed as requested.

File: README.md (hunk 1 above)
Comment (prcorcoran) [reply README.md:7]: Fixed.

**Comment by Ledespana** [comment]
Please remove me from your mailing list.
Thanks

**Comment by gary-moon** [comment]
> Please remove me from your mailing list.
> Thanks

You will need to do that yourself. Click the `Watch` button and change your notification settings (or unwatch the project).

image

____

In addition, make sure to unsubscribe using the button on the right side of this page.

image

File: README.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ License URI: https://opensource.org/licenses/GPL-2.0
 
 ## Building
 
-$ zip -r v2.0.0.0.zip ./localiq_joomla_30_extension  -x *.git*
+$ zip -r localiq_plugin_v2.1.0.0.zip ./localiq_joomla_30_extension -x ./localiq_joomla_30_extension/.git/\*
```
Comment (gary-moon) [inline README.md]: The project name has been changed. Should we update the directory name here to match it?

Comment (prcorcoran) [reply README.md]: Yes, done.

**Review by gary-moon — APPROVED** [review]
👍

**Pull request merged**